robotoff/insights/importer.py

```diff
@@ -1475,6 +1475,35 @@ def compute_crop_bounding_box(
         return results
 
 
+class IngredientSpellcheckImporter(InsightImporter):
+
+    @staticmethod
+    def get_type() -> InsightType:
+        return InsightType.ingredient_spellcheck
+    
+    @classmethod
+    def get_required_prediction_types(cls) -> set[PredictionType]:
+        return {PredictionType.ingredient_spellcheck}
+
+    @classmethod
+    def generate_candidates(
+        cls,
+        product: Optional[Product],
+        predictions: list[Prediction],
+        product_id: ProductIdentifier,
+    ) -> Iterator[ProductInsight]:
+        # Only one prediction
```

Comment:
Possibly more than 1, no?

Reply:
There's only one correction, no?
Or maybe I didn't understand what it means